BXMSDOC-7115 Swagger UI and OAS URLs revision following code change (#3214)
kiegroup/kie-docs: doc-content/kogito-docs/src/main/asciidoc/bpmn/chap-kogito-developing-process-services.adoc

Based on these processes and on application configurations, {PRODUCT} generates a set of REST operations to create new orders, to list and delete active orders, and to determine the age of a specified person.

> For example, the following REST operations use the endpoint `/orders` to interact with customer orders. You can use a REST client, curl utility, or the Swagger UI configured for the application (ex: \http://localhost:8080/q/swagger-ui or \http://localhost:8080/swagger-ui.html etc. depending on configuration) to send API requests to interact with the running application.

Reviewer: ```suggestion
For example, the following REST operations use the endpoint `/orders` to interact with customer orders. You can use a REST client, curl utility, or the Swagger UI configured for the application (such as \http://localhost:8080/q/swagger-ui or \http://localhost:8080/swagger-ui.html) to send API requests to interact with the running application.
```

> .Swagger UI to interact with all application endpoints (ex: \http://localhost:8080/q/swagger-ui or \http://localhost:8080/swagger-ui.html etc. )

Reviewer: ```suggestion
.Swagger UI to interact with all application endpoints (such as \http://localhost:8080/q/swagger-ui or \http://localhost:8080/swagger-ui.html)
```

Reviewer: And so on for the others as well.

Author: > And so on for the others in this PR as well.

Done 👍
